scipy/stats/stats.py

```diff
@@ -4541,6 +4542,131 @@ def ttest_rel(a, b, axis=0, nan_policy='propagate'):
     return Ttest_relResult(t, prob)
 
 
+def ttest_trimmed(a, b, axis=0, equal_var=False, nan_policy='propagate', trimming_percentage=20.0):
```

Comment:
does equal_var have any effect in the code below?

Reply:
Only when performing a regular t-test when the nan_policy is "omit", but that needs to be changed anyway.
The author wrote two papers - one in which variances are assumed equal and one in which they are not. So this should treat both cases.